addons/main/stringtable.xml

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="SSS">
+	<Package name="Main">
+		<Key ID="STR_SSS_Main_SimplexSupportService">
+			<English>Simplex Support Services</English>
+			<French>Simplex Support Services</French>
+		</Key>
+		<Key ID="STR_SSS_Main_Setting_AdminFullAccess_DisplayName">
+			<English>Give admins access to all supports</English>
+			<French>Donner un accès complet aux admins</French>
+		</Key>
+		<Key ID="STR_SSS_Main_Setting_AdminFullAccess_Description">
+			<English>Admins will be able to use every support available</English>
+			<French>Les admins seront capable d'utiliser tous les supports disponibles</French>
+		</Key>
+		<Key ID="STR_SSS_Main_Setting_AdminLimitSide_DisplayName">
+			<English>Limit admin access to side</English>
+			<French>Limiter l'accès admin à la faction</French>
+		</Key>
+		<Key ID="STR_SSS_Main_Setting_AdminLimitSide_Description">
+			<English>Limit the admin access to the current side of the admin</English>
+			<French>Limite les accès admin aux supports de la faction de l'admin</French>
+		</Key>
+		<Key ID="STR_SSS_Main_Setting_GiveUAVTerminal_DisplayName">
+			<English>Give UAV Terminal on drone request</English>
+			<French>Fournir un terminal pour les requêtes de drone</French>
+		</Key>
+		<Key ID="STR_SSS_Main_Setting_GiveUAVTerminal_Description">
+			<English>Gives CAS Drone requesters a UAV terminal if they don't have one</English>
+			<French>Fournit un terminal de controle s'il n'en possède pas au joueur capable de faire des requêtes de drone</French>
+		</Key>
+		<Key ID="STR_SSS_Main_Setting_DirectActionRequirement_DisplayName">
+			<English>Require access item/condition for direct action</English>
+			<French>Nécessite une dotation/une condition spécifique</French>
+		</Key>
+		<Key ID="STR_SSS_Main_Setting_DirectActionRequirement_Description">
+			<English>When disabled, anyone can interact directly with transports or logistics booths</English>
```

Comment:
When disabled, allow anyone to interact directly with transports or logistics booths